clients/src/main/java/org/apache/kafka/common/config/internals/BrokerSecurityConfigs.java

```diff
@@ -34,6 +34,7 @@
     public static final String SSL_CLIENT_AUTH_CONFIG = "ssl.client.auth";
     public static final String SASL_ENABLED_MECHANISMS_CONFIG = "sasl.enabled.mechanisms";
     public static final String SASL_SERVER_CALLBACK_HANDLER_CLASS = "sasl.server.callback.handler.class";
+    public static final String CONNECTIONS_MAX_REAUTH_MS = "connections.max.reauth.ms";
```

Comment:
is it going to be used for sasl only if so can you make it "sasl.connection.max.reauth.ms"

Reply:
@rajinisivaram I believe there was a desire to keep our options open in this regard and not mention SASL explicitly -- thoughts?

Reply:
Yes, since this config is used by the broker to force reauthentication using connection termination as well, there is no reason why we can't apply it for SSL as well in future. So it makes sense to keep it neutral.